modules/subchannel/doc/content/modules/subchannel/general/subchannel_theory.md

```diff
@@ -269,14 +268,14 @@ This is the default algorithm, where the unknown flow variables are calculated i
 
 #### Implicit segregated
 
-In this case, the governing equations are recast in matrix form and the flow variables are calculated by solving the corresponding system. This means that variables are retrieved concurrently for the whole block. Otherwise, the solution algorithm is the same as in the default method.
+In this case, the governing mass, axial momentum and crossflow momentum, equations are recast in matrix form and the flow variables are calculated by solving the corresponding system. This means that variables are retrieved concurrently for the whole block. Otherwise, the solution algorithm is the same as in the default explicit method.
 
 #### Implicit
 
-In this case, the conservation equations are recast in matrix form and combined into a single system. The user can decide whether or not they will include the enthalpy calculation in the matrix formulation. The flow variables are calculated by solving that big system to retrieve all the unknows at the same time instead of one by one, and on all the nodes of the block: $\vec{\dot{m}}, \vec{P}, \vec{w_{ij}}, \vec{h}$. The solution algorithm is the same as in the default method, but the solver used in this version is a fixed point iteration instead of a Newton method. The system looks like this:
+In this case, the governing mass, axial momentum and crossflow momentum  conservation equations are recast in matrix form and combined into a single system. The system of all the subchannel equations looks like this:
 
 \begin{equation}
-\begin{bmatrix}
+\begin{bmatrix
```

Comment:
```suggestion
\begin{bmatrix}
```

please check the website rendering when changing doco
https://mooseframework.inl.gov/docs/PRs/31785/site/modules/subchannel/general/subchannel_theory.html